src/versionUtils.ts

```diff
@@ -197,7 +198,20 @@ function setVersionAndTellUser(msg: string, newVersion: string) {
 	extension.setRuntimeVersionSetting(newVersion);
 }
 
-export async function getDownloadURLForCamelKTag(camelKVersion : string, platformStr : string): Promise<string> {
+function toKamelOsString(platform: Platform | undefined): string | undefined {
+	switch (platform) {
+		case Platform.WINDOWS:
+			return 'windows';
+		case Platform.LINUX:
+			return 'linux';
+		case Platform.MACOS:
+			return 'mac';
+	}
+	return undefined;
+}
+
+export async function getDownloadURLForCamelKTag(camelKVersion : string, platform : Platform): Promise<string> {
+	let platformStr = toKamelOsString(platform);
```

Comment:
optional: better to use const as the value is nto changed after

Reply:
Right!